diracx-testing/src/diracx/testing/__init__.py

```diff
@@ -451,6 +451,50 @@ def with_config_repo(tmp_path_factory):
                 }
             },
             "Operations": {"Defaults": {}},
+            "Systems": {
+                "WorkloadManagement": {
+                    "Production": {
```

Comment:
```suggestion
```

Reply:
do I remove the entire key?

Reply:
Where will `MaxRescheduling` be now, do you think?

Reply:
> do I remove the entire key?

The key `Production` should be removed, and its value should be associated directly to the `WorkloadManagement` key